### DISCUSSION TITLE
Deploy文檔中的InstallUiautomator2是必須的嗎

### DISCUSSION THREAD

**Discussion by bookbug666** [LmeSzinc/AzurLaneAutoScript#2207] [Q&A] [answered]
使用別的截圖/控制方式還需要嗎，一早起來看到alas卡在安裝但我模擬器本就裝了....

**Comment by LmeSzinc** [comment, marked as answer]
是的，安卓上的 minitouch, DroidCast, DroidCast_raw 服务都是由 ATX-agent 启动并守护的，也就是模拟器里面的小黄车图标的应用，module/device/method 下对应的类也继承了 Uiautomator2。如果不用 atx 的话，alas 需要自己搓一个类似 atx 的东西。
